rss/serializers/atom_1_0.py

```diff
@@ -1,15 +1,81 @@
 """Serializer for Atom 1.0"""
 
-from flask.json import jsonify
+from feedgen.feed import FeedGenerator
+from flask import url_for
 from typing import Tuple, Optional
 from arxiv import status
 from rss.serializers.serializer import Serializer
+from rss.serializers.atom_extensions import ArxivExtension, ArxivEntryExtension
 from elasticsearch_dsl.response import *
 
+import datetime
+import pytz
+
 
 class Atom_1_0(Serializer):
```

Comment:
This class and methods could use some docstrings as well. I haven't  looked at other files not in this PR, but really we are aiming for just about every class and method to have a docstring.